### PR TITLE
fix(#2507): icon in message after rename-file

### LINKED ISSUE #2507: Icon in message after rename file
Hi.
Thanks for your plugin.

This is not important but it is for no nvim-web-devicons.

I suggest changing rename-file.lua.

before 

스크린샷 2023-11-02 08-25-04

after

스크린샷 2023-11-02 08-25-15

### PR DESCRIPTION
**Pull request by Dongyomi**
Closes #2507

### CODE REVIEW

**Comment by Akmadan23** [comment]
FYI: you don't need to open a new PR when you want to change something, you can just add another commit to the branch.

**Comment by Dongyomi** [comment]
ok thank you. should i close PR and commit again?

**Comment by Akmadan23** [comment]
No, if some changes are needed just push a new commit to your `test` branch.

File: lua/nvim-tree/actions/fs/rename-file.lua [hunk 1]
```diff
@@ -53,7 +53,9 @@ function M.fn(default_modifier)
 
     -- support for only specific modifiers have been implemented
     if not ALLOWED_MODIFIERS[modifier] then
-      return notify.warn("Modifier " .. vim.inspect(modifier) .. " is not in allowed list : " .. table.concat(ALLOWED_MODIFIERS, ","))
+      return notify.warn(
+        string.format("Modifier %s is not in allowed list: %s", vim.inspect(modifier), table.concat(ALLOWED_MODIFIERS, ","))
```
Comment (Akmadan23) [inline lua/nvim-tree/actions/fs/rename-file.lua:57]: As @gegoune said in #2508, this is a change outside of the scope of your PR. Using `string.format` here is arguably better, but it should be done in a separate refactoring.

Comment (Dongyomi) [reply lua/nvim-tree/actions/fs/rename-file.lua:57]: I am sorry. I understood gegoune's words to fix that as well.

Comment (Akmadan23) [reply lua/nvim-tree/actions/fs/rename-file.lua]: Well, now that I think about it, I'm not sure if the "same here" is about the unrelated style changes or the use of `string.format`... Let's wait for his feedback.

Comment (Dongyomi) [reply lua/nvim-tree/actions/fs/rename-file.lua]: Okay and I finally understand your FYI. Thank you for your kindness.

Comment (gegoune) [reply lua/nvim-tree/actions/fs/rename-file.lua]: Apologies if I wasn't clear enough. My fault. 

Regarding unrelated changes: I meant it for formatting changes toward the top of file. 

`string.format` is what we should be using going forward but as @Akmadan23 said existing code should not be changed as part of that pr to make it self contained. We can replace other string concatenations across code base in separate pr (I can do it). 

Hope that clears things up. Thanks for your pr!

**Review by alex-courtis — APPROVED** [review]
Many thanks for your contribution!

Compatibility with all terminals is desirable.

**Pull request merged**